templates/taxi/car_detail.html

```diff
@@ -12,6 +12,16 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  {% if user not in car.drivers.all  %}
```

Comment:
The condition `{% if user not in car.drivers.all %}` should use parentheses to call the `all()` method: `{% if user not in car.drivers.all() %}`. This ensures the method is executed and returns a queryset.